swift/src/Ice/CurrentExtensions.swift

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) ZeroC, Inc.
+
+import Foundation
+
+extension Current {
+    /// Creates an outgoing response with reply status `ok`.
+    /// - Parameters:
+    ///   - result: The result to marshal into the response payload.
+    ///   - formatType: The class format.
+    ///   - marshal: The action that marshals result into an output stream.
+    /// - Returns: The outgoing response.
+    public func makeOutgoingResponse<T>(_ result: T, formatType: FormatType, marshal: (OutputStream, T) -> Void)
```

Comment:
In C# we named these methods `createXxx`, in C++ is like in swift `makeXxx`.  Is the difference intentional?

Reply:
Yes, the convention for factory methods in Swift is makeXxx.